src/App.svelte

```diff
@@ -0,0 +1,216 @@
+  let noteIconImage = "./images/sticky-note-regular.svg";
+  let calendarIconImage = "./images/calendar.svg";
+  let dateString;
+  let isEditing = false;
+  let xxx = "";
+
+  let now = new Date(),
+    month,
+    day,
+    year;
+
+  onMount(() => {
+    db.collection("grocery").onSnapshot((snapData) => {
+      groceryList = snapData.docs;
+    });
+    (month = "" + (now.getMonth() + 1)),
+      (day = "" + now.getDate()),
+      (year = now.getFullYear());
+    if (month.length < 2) month = "0" + month;
+    if (day.length < 2) day = "0" + day;
+    dateString = [year, month, day].join("-");
+  });
+
+  function removeItem(itemId) {
+    console.log("The event works");
+    console.log(itemId);
+    db.collection("grocery").doc(itemId).delete();
+  }
+
+  // async function updateItem(itemId, newItemValue) {
+  //    const response = await db.collection("grocery").doc(itemId).update({ name: newItemList[itemId] });
+  //   newItemList[itemId] = '';
+  //   isEditing = false;
+
+  // }
+
+  function editItem() {
+    isEditing = true;
+  }
+
+  function updateItem(itemId, inputEl) {
+    console.log(itemId);
+    console.log(inputEl);
+    console.log(newItemList);
+    db.collection("grocery").doc(itemId).update({ name: inputEl });
+    console.log("no update");
+    isEditing = false;
+
+    // console.log(isEditing);
+    // isEditing = false;
+  }
+</script>
+
+<header />
+<main>
+  <h1>Grocery list</h1>
+  <h3>
+    {dateString} <img src={calendarIconImage} alt="{calendarIconImage} image" />
+  </h3>
+  <AddItemForm />
+
+  <ul>
+    {#each groceryList as listItem}
+      <GroceryItem
+        item={listItem.data()}
+        {removeItem}
+        itemId={listItem.id}
+        {isEditing}
+        {updateItem}
+        inputEl={listItem.data().name}
+        {editItem}
+      />
+    {/each}
+  </ul>
+  <div class="bottom-navbar">
+    <button class="bottom-navbar-btn share "
+      >Share<img
+        class="icon share"
+        src={shareIconImage}
+        alt="{shareIconImage} image"
+      /></button
+    >
+    <button class="bottom-navbar-btn"
+      >Set Reminder<img
+        class="icon"
+        src={bellIconImage}
+        alt="{bellIconImage} image"
+      /></button
+    >
+    <button class="bottom-navbar-btn"
+      >Make recurring<img
+        class="icon"
+        src={redoIconImage}
+        alt="{redoIconImage} image"
+      /></button
+    >
+    <button class="bottom-navbar-btn"
+      >Add note<img
+        class="icon"
+        src={noteIconImage}
+        alt="{noteIconImage} image"
+      /></button
+    >
+  </div>
+</main>
+<footer />
+
+<style>
+  main {
+    text-align: center;
+    padding: 1em;
+    margin: 0 auto;
+    /*background-color: var(--background-color-dark);*/
+    background-color: gainsboro;
+    border-radius: 5px;
+  }
+  h1 {
+    color: var(--title-color);
+    font-size: 4em;
+    font-weight: 100;
+  }
+  ul {
+    list-style-type: none;
+    width: 350px;
```

Comment:
The page breaks on mobile (320px), it's missing here a `max-width` to make sure it's fully responsive. [see screenshot](https://user-images.githubusercontent.com/14869087/112058382-72879780-8b52-11eb-9360-cccd540d1cb5.png)

Reply:
Thanks for this feedback. I was wondering why you talked about the mobile size of 320px and I realized that It is now the smallest mobile size:) .